CONTRIBUTING.md

```diff
@@ -187,16 +188,18 @@ This is a short way for you to say that you are entitled to contribute the patch
 It is a legal statement that asserts your agreement with the [DCO](#developers-certificate-of-origin).
 Therefore, it must have your *real name* (First Last) and a valid e-mail address.
 Adding this tag can be done automatically by using `git commit -s`.
-If you are editing files and committing through GitHub, you must write your real name in the “Name” field in GitHub settings and the email used in the "Signed-off-by:" must be your primary github email.
-You must manually add the "Signed-off-by:" to the commit message that github requests.
+If you are editing files and committing through GitLab, you must write your real name in the “Full Name” field in your GitLab profile and the email used in the "Signed-off-by:" must be your "Commit email" address.
+You must manually add the "Signed-off-by:" to the commit message that gitlab requests.
 If you are editing files and committing on your local PC, set your name and email with:
 
 ```bash
 git config --global user.name "my name"
 git config --global user.email "my@email.address"
 ```
 
-### Pull Request workflow
+Then, adding the "Signed-off-by" line is as simple as using `git commit -s` instead of `git commit` (using an alias is recommended, e.g. `git config --global alias.ci='commit -s'`)
```

Comment:
```suggestion
Then, adding the "Signed-off-by" line is as simple as using `git commit -s` instead of `git commit` (using an alias is recommended. E.g. `git config --global alias.ci='commit -s'`)
```